src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -77,15 +77,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $io = new SymfonyStyle($input, $output);
         $filenames = $input->getArgument('filename');
+        $hasStdin = '-' === ($filenames[0] ?? '');
 
-        if (0 === \count($filenames)) {
-            if (0 === ftell(STDIN)) {
-                $template = '';
-                while (!feof(STDIN)) {
-                    $template .= fread(STDIN, 1024);
+        if ($hasStdin || 0 === \count($filenames)) {
+            if ($hasStdin || 0 === ftell(STDIN)) { // remove 0 === ftell(STDIN) check in 5.0
+                if (!$hasStdin) {
+                    @trigger_error('Calling to the "lint:twig" command providing pipe file content to STDIN without passing the dash symbol "-" explicitly is deprecated since Symfony 4.4.', E_USER_DEPRECATED);
```

Comment:
I think we usually also display a deprecation notice in the command output, because the silenced notice maybe be not visible enough. Please check past deprecations in command lines :)

Reply:
They are displayed in the command output only when we use very verbose mode `-vv` or debug `-vvv`.

Reply:
do we consider this is enough? in 2.8 we used some `$io->caution()` but I can't find any in 3.4, so maybe :)

unrelated: we need to update the description of the commands:
A file, a directory or "-" for reading from STDIN

Reply:
> do we consider this is enough? in 2.8 we used some $io->caution() but I can't find any in 3.4, so maybe :)

I'm not sure why we stopped to do that, but maybe we always should display them if debug mode is enabled as well? note that for apps with monolog enabled the deprecation notice will be included in logs files.

Anyway, that would merit a separate PR for all commands that are currently displaying a deprecation warning.

> unrelated: we need to update the description of the commands:
A file, a directory or "-" for reading from STDIN

done.